Update active version information for 2020 (#679)
RoboCupAtHome/RuleBook: introduction/Infrastructure.tex

\subsection{RoboCup@Home Mailinglist}
The official \iterm{RoboCup@Home mailing list} can be reached at
\begin{center}
\href{mailto:robocup-athome@lists.robocup.org}{\texttt{robocup-athome@lists.robocup.org}}
\end{center}
You can register to the email list at:
\begin{center}
{\small\url{http://lists.robocup.org/cgi-bin/mailman/listinfo/robocup-athome}}
\end{center}

\subsection{RoboCup@Home Web Page}
The official \iterm{RoboCup@Home website} that also hosts this RuleBook can be found at:
\begin{center}
{\small\url{https://athome.robocup.org/}}
\end{center}

\subsection{RoboCup@Home Rulebook Repository}

> The official \iterm{RoboCup@Home Rulebook Repository} is where rules are publicly discussed before applying changes, buffs, and nerfs to the rulebook.

Reviewer: ```suggestion
The official \iterm{RoboCup@Home Rulebook Repository} is where rules are publicly discussed before applying changes to the rulebook.
```

Buffs is informal, and nerfs isn't a word. Let's just leave it at any changes.

> The entire RoboCup@Home community is welcome and encouraged to actively participate in rule discussion and crafting. The repository can be reached at:

Reviewer: ```suggestion
The entire RoboCup@Home community is welcome and encouraged to actively participate in creating and discussing the rules. The repository can be reached at:
```

I don't like the word crafting. I used **google** `define: create` and `define: crafting`

```
cre·ate
/krēˈāt/
verb
gerund or present participle: creating
bring (something) into existence.
```

```
craft·ing
/ˈkraftiNG/
noun
the activity or hobby of making decorative articles by hand.
```

Author: Time to go back to school and learn English again...

\begin{center}
{\small\url{https://github.com/RoboCupAtHome/RuleBook/}}
\end{center}

Although opening issues with inconsistencies, questions, clarifications, and suggestions is highly appreciated, the best way to contribute is by making pull requests with fixes and proposed changes.

\subsection{RoboCup@Home Telegram Group}

> The official \iterm{RoboCup@Home Telegram Group} is and communication channel for the RoboCup@Home community where rules are discussed, announcements are made, and questions are answered.

Reviewer 2: Small typo:  The official \iterm{RoboCup@Home Telegram Group} is **a** communication

Reviewer: ```suggestion
The official \iterm{RoboCup@Home Telegram Group} is a communication channel for the RoboCup@Home community where rules are discussed, announcements are made, and questions are answered.
```

> However, beyond the technical aspects of the competition, the \textit{Telegram Group} is a meeting point to stay in contact with the community, foster knowledge exchange and strengthen relationships.

Reviewer 2: Maybe: In addition to discussing technical aspects of the competition, ...

However sounds like this is opposed to what was stated before.

Author: Any suggestions?

Reviewer 3: ```suggestion
Beyond the technical aspects of the competition, the \textit{Telegram Group} is a meeting point to stay in contact with the community, foster knowledge exchange and strengthen relationships.
```
